test/tests/functional/pbs_qmgr.py

```diff
@@ -47,6 +47,55 @@ class TestQmgr(TestFunctional):
     Test suite for PBSPro's qmgr command
     """
 
+    def setup_sc1(self):
+        a = {'partition': 'P1,P4',
+             'sched_host': self.server.hostname,
+             'sched_port': '15050'}
+        self.server.manager(MGR_CMD_CREATE, SCHED,
+                            a, id="sc1")
+        self.scheds['sc1'].create_scheduler()
+        self.scheds['sc1'].start()
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 'True'}, id="sc1", expect=True)
+
+    def setup_sc2(self):
+        dir_path = '/var/spool/pbs/sched_dir'
+        if not os.path.exists(dir_path):
+            self.du.mkdir(path=dir_path, sudo=True)
+        a = {'partition': 'P2',
+             'sched_priv': os.path.join(dir_path, 'sched_priv_sc2'),
+             'sched_log': os.path.join(dir_path, 'sched_logs_sc2'),
+             'sched_host': self.server.hostname,
+             'sched_port': '15051'}
+        self.server.manager(MGR_CMD_CREATE, SCHED,
+                            a, id="sc2")
+        self.scheds['sc2'].create_scheduler(dir_path)
+        self.scheds['sc2'].start(dir_path)
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 'True'}, id="sc2", expect=True)
+
+    def setup_sc3(self):
+        a = {'partition': 'P3',
+             'sched_host': self.server.hostname,
+             'sched_port': '15052'}
+        self.server.manager(MGR_CMD_CREATE, SCHED,
+                            a, id="sc3")
+        self.scheds['sc3'].create_scheduler()
+        self.scheds['sc3'].start()
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 'True'}, id="sc3", expect=True)
+
+    def setup_sc4(self):
+        a = {'partition': 'P5',
+             'sched_host': self.server.hostname,
+             'sched_port': '15053'}
+        self.server.manager(MGR_CMD_CREATE, SCHED,
+                            a, id="sc4")
+        self.scheds['sc4'].create_scheduler()
+        self.scheds['sc4'].start()
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 'True'}, id="sc4", expect=True)
```

Comment:
If you are going to copy all this code over from the multisched test plan, just add your test to the multisched test plan.  It belongs there more than here anyway.